### DISCUSSION TITLE
How to generate the input files of ww3_bounc with wave spectra?

### DISCUSSION THREAD

**Discussion by Rachel201810** [NOAA-EMC/WW3#867] [Q&A] [answered]
Dear All,

Hi,
How to generate the input files of ww3_bounc/ww3_bound (File(s) with wave spectra) for running a regional case?
I can't access the website listed in mamual as "ftp://polar.ncep.noaa.gov/pub/waves/develop/glw.latest_run/
http://tinyurl.com/iowagaftp/HINDCAST/GLOBAL/2009_ECMWF/SPEC".

Thanks!

**Comment by aliabdolali** [comment, marked as answer]
@Rachel201810 see https://github.com/NOAA-EMC/WW3/discussions/706
There are a few things you need to do:
1) run a global simulation and define the coordinates of open boundary nodes of the regional grid in the global setup and then generate directional spectral density files using ww3_ounp (for netcdf) or ww3_outp (for ascii).  Type 1 subtype 3 is the one you need to use. 
Or, you can use the directional spectral density taken from a global setup along your open boundary nodes, however you need to make sure  the spectral and directional resolution of your boundary conditions match the regional configuration.
2) in your ww3_grid.inp(nml) of the regional grid (mesh), define the open boundary nodes (see regtests/ww3_tp2.17/input/ww3_grid_a.inp).
3) Use ww3_bound for ascii or ww3_bounc for netcdf spec files and generate nest.ww3 (see regtests/ww3_tp2.17/input/ww3_bounc.inp).

**Reply by Rachel201810** [reply]
Thank you very much for your help!
